docs/interfaces/auth-service.txt

```diff
@@ -0,0 +1,45 @@
+Interface Token {
+    payload: {
+        // The lifetime of the Token in milliseconds.
+        expiresIn: number (ms),
```

Comment:
```suggestion
        expiresAt: number, // the date in unix seconds of the expiration.
```
If you use `lifetime`, the refrence is not clear. E.g. the token expires in two minutes. But from where?

Reply:
@GabrielInTheWorld I think this is not resolved. Please give me a call if something is not clear.